app/views/notes/show.html.erb

```diff
@@ -2,6 +2,7 @@
 
 <div class="col-md-9 note-show<% if @node.status == 4 || @node.status == 0 %> moderated<% end %>">
   <% if current_user && @node.tags.length == 0 %><div class="alert alert-warning"><%= raw t('notes.show.note_no_tags') %></div><% end %>
+  <% if @node.has_power_tag('status:candidate') %><div class="alert alert-info"><a href="https://publiclab.org/notes/warren/09-17-2016/what-makes-a-good-activity" >read about how to adapt its content</a></div>
```

Comment:
Ah - actually we should have this whole line -- sorry i wasn't specific enough and thank you!

> This was marked as a candidate activity -- [read about how to adapt its content]() into a fully-fledged, step-by-step activity that can be easily replicated.